src/OrchardCore/OrchardCore.Users.Core/Services/UserStore.cs

```diff
@@ -524,5 +531,106 @@ public async Task<IList<IUser>> GetUsersForClaimAsync(Claim claim, CancellationT
         }
 
         #endregion IUserClaimStore<IUser>
+
+        #region IUserAuthenticationTokenStore
+        public Task<string> GetTokenAsync(IUser user, string loginProvider, string name, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (user == null)
+            {
+                throw new ArgumentNullException(nameof(user));
+            }
+
+            if (string.IsNullOrEmpty(loginProvider))
+            {
+                throw new ArgumentException(nameof(loginProvider));
```

Comment:
@ArieGato FYI, this overload takes an exception message, not just a parameter name like `ArgumentException`. You'll probably want something like this:

```suggestion
                throw new ArgumentException("The login provider cannot be null or empty.", nameof(loginProvider));
```

Same for the other occurrences.